Sources/UInt128.swift

```diff
@@ -435,7 +442,7 @@ extension UInt128 : BinaryInteger {
 
         var words: [UInt] = []
 
-        for currentWord in 0 ... self.bitWidth / UInt.bitWidth {
+        for currentWord in 0 ..< self.bitWidth / UInt.bitWidth {
```

Comment:
Was the `...` incorrect? If it was, I’d like to see a test capturing this so it doesn’t crop up again.